evaluation-tasks/requirements.txt

```diff
@@ -0,0 +1,9 @@
+boto3
+luigi
+numpy
+pre-commit
+requests
+soundfile
+python-slugify
+tqdm
+
```

Comment:
Similar to the above, but do we want to treat this whole repo as one Python package?

Reply:
Ya I like that.